qa/rpc-tests/getblocktemplate.py

```diff
@@ -49,11 +49,15 @@ def run_test(self):
 
         # Test 5: General checks
         tmpl = node.getblocktemplate()
-        assert(len(tmpl['noncerange']) == 16)
+        assert_equal(16, len(tmpl['noncerange']))
```

Comment:
What's the difference between these two asserts?

Reply:
assert_equals is essentially just a wrapper for assert. I think it reads a bit better when we are not just checking a boolean value.